src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1124,10 +1126,10 @@ sub check_star_catalog {
 	# Faint and bright limits ~ACA-009 ACA-010
 	if ($type ne 'MON' and $mag ne '---') {
 
-	    if ($mag < $mag_bright or $mag > $mag_faint_red) {
+	    if ($mag < $mag_bright or $mag > $self->{mag_faint_red}) {
```

Comment:
Did we decide in the working group to write out the mag limit that was used in the warning?  Or to put it in the block that includes the probabilities.